### ISSUE TITLE
 Rule Proposal: Always use :focus when using :hover

### ISSUE THREAD

**Issue by mgifford** [stylelint/stylelint#3307]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

New rule to check for an accessibility problem that is common in CSS. 

> Which rule, if any, is this issue related to?

??

> What CSS is needed to reproduce this issue?

```
a:hover {
    color:red;
}
```

> What stylelint configuration is needed to reproduce this issue?

??

> Which version of stylelint are you using?

Where is this located?

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

??

> What did you expect to happen?

It should issue a warning and tell the user to  use:

```
a:hover,
a:focus {
    color:red;
}
```

> What actually happened (e.g. what warnings or errors you are getting)?

??

**Comment by evilebottnawi** [comment]
It looks like you just deleted our lovely crafted issue template. It was there for good reasons. Please help us solving your issue by answering the questions asked in this template. I'm closing this. Please either update the issue with the template and reopen, or open a new issue.

**Comment by mgifford** [comment]
With a name like @evilebottnawi I read that as "Evil Bot Nawi"

Here's the WCAG 2 recommendation:
https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus

Here's one from CSS Lint:
https://github.com/CSSLint/csslint/issues/264

Here's some more stuff about proper use of focus:
https://www.deque.com/blog/give-site-focus-tips-designing-usable-focus-indicators/

**Comment by evilebottnawi** [comment]
@mgifford please add this all information to original issue and reopen issue

**Comment by mgifford** [comment]
This might well be a duplicate of https://github.com/stylelint/stylelint/issues/2650

**Comment by ntwb** [comment]
Nope, this is unacceptable @mgifford, making personal attacks on members of the stylelint team is unacceptable, you are no longer welcome here at stylelint.
